qutebrowser/browser/adblock.py

```diff
@@ -59,6 +61,24 @@ def get_fileobj(byte_io):
     return io.TextIOWrapper(byte_io, encoding='utf-8')
 
 
+def is_whitelisted_domain(host):
+    """Check if the given host is on the adblock whitelist.
+
+    Args:
+        host: The host as given by the adblocker as string.
+    """
+    whitelist = objreg.get('config').get('content', 'host-blocking-whitelist')
```

Comment:
You can just do `from qutebrowser.config import config` and then use `config.get`